README.md

```diff
@@ -1 +1,108 @@
 # "Ostia" Project main repository
+
+Ostia aims to be an openshift native API Management software.
+
+This is a not supported/official redhat product.
+
+## Deployment
+
+* To learn how to deploy openshift locally:
+
+<https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md>
+
+* Clone the repo:
+
+```
+git clone https://github.com/3scale/ostia.git
```

Comment:
I did:

```shell
go get github.com/3scale/ostia
cd "$GOPATH/src/github.com/3scale/ostia"
```

Reply:
That's how to deploy "ostia", you don't need golang.

Reply:
True. Good point. Then this applies only to the git cloning in the development section.